railties/lib/rails/generators/actions.rb

```diff
@@ -307,6 +307,7 @@ def execute_command(executor, command, options = {}) # :doc:
           config = { verbose: false }
 
           config[:capture] = options[:capture] if options[:capture]
+          config[:abort_on_failure] = options[:abort_on_failure] if options[:abort_on_failure]
```

Comment:
Would this not require thor 0.20.3?

Reply:
Absolutely, :+1:. We should either check the loaded version of `thor` if the option is passed, and give a proper error if not implemented in the loaded version, or bump the minimum thor requirement.

Reply:
Since this is only going to be released in 6.0. I'd bump the requirement.

Reply:
I'm wondering, since this is only going to be released in 6.0, should we actually change the default? In some cases `abort_on_failure: false` may be want you want, but in the general case, I think users expect the option to be `true`.

Reply:
For now I bumped the dependency, added some docs, and a changelog entry!